resolwe_bio/processes/workflows/bbduk_star_featurecounts_qc.py

```diff
@@ -0,0 +1,802 @@
+                )
+                out_sam_attributes = StringField(
+                    label="Desired SAM attributes [--outSAMattributes]",
+                    default="Standard",
+                    choices=[
+                        ("Standard", "Standard"),
+                        ("All", "All"),
+                        ("NH HI NM MD", "NH HI NM MD"),
+                        ("None", "None"),
+                    ],
+                    description="A string of desired SAM attributes, in the order desired for the "
+                    "output SAM.",
+                )
+                out_rg_line = StringField(
+                    label="SAM/BAM read group line [--outSAMattrRGline]",
+                    required=False,
+                    description="The first word contains the read group identifier and must start "
+                    "with ID:, e.g. --outSAMattrRGline ID:xxx CN:yy ”DS:z z z” xxx will be added as "
+                    "RG tag to each output alignment. Any spaces in the tag values have to be double "
+                    "quoted. Comma separated RG lines correspons to different (comma separated) input "
+                    "files in –readFilesIn. Commas have to be surrounded by spaces, e.g. "
+                    "–outSAMattrRGline ID:xxx , ID:zzz ”DS:z z” , ID:yyy DS:yyyy.",
+                )
+
+            chimeric_reads = GroupField(
+                ChimericReadsOptions,
+                label="Chimeric reads options",
+                hidden="!alignment.show_advanced",
+            )
+            transcript_output = GroupField(
+                TranscriptOutputOptions,
+                label="Transcript coordinate output options",
+                hidden="!alignment.show_advanced",
+            )
+            filtering_options = GroupField(
+                FilteringOptions,
+                label="Output filtering options",
+                hidden="!alignment.show_advanced",
+            )
+            alignment_options = GroupField(
+                AlignmentOptions,
+                label="Alignment options",
+                hidden="!alignment.show_advanced",
+            )
+            two_pass_options = GroupField(
+                TwoPassOptions,
+                label="Two-pass mapping options",
+                hidden="!alignment.show_advanced",
+            )
+            output_options = GroupField(
+                OutputOptions, label="Output options", hidden="!alignment.show_advanced"
+            )
+
+        class Quantification:
+            """Quantification (featureCounts)."""
+
+            annotation = DataField(
+                "annotation",
+                label="Annotation",
+                description="GTF and GFF3 annotation formats are supported",
+            )
+            show_advanced = BooleanField(
+                label="Advanced options",
+                default=False,
+            )
+            assay_type = StringField(
```

Comment:
I'm in favour of exposing `assay_type` and related `cdna_index` inputs on the top-level, as this is a critical parameter that must not be selected in the wrong way (it must match the library type of the dataset being analysed). If the setting is hidden in the advanced section of the inputs, it is easy to get this wrong.